tests/trx_generator/main.cpp

```diff
@@ -24,33 +24,40 @@ enum return_codes {
 };
 
 int main(int argc, char** argv) {
-   const int64_t TRX_EXPIRATION_MAX = 3600;
+   const int64_t trx_expiration_max = 3600;
+   const uint16_t generator_id_max = 960;
```

Comment:
Should these be `constexpr`?

Reply:
Like @huangminghuang said, they could be `constexpr` but don't necessarily need to be due to the limited use and not being used in generic programming.  I can still change them to `constexpr` if you'd like. Just let me know.

Reply:
I was unsure of our standard. If it's not "constexpr all the things" then there is no reason to change. Marking resolved.